test/test_utils/sagemaker.py

```diff
@@ -53,14 +57,31 @@ def assign_sagemaker_remote_job_instance_type(image):
 
 
 def assign_sagemaker_local_job_instance_type(image):
-    if "tensorflow" in image and "inference" in image and "gpu" in image:
+    if "graviton" in image:
+        return "c6g.2xlarge"
+    elif "tensorflow" in image and "inference" in image and "gpu" in image:
         return "g4dn.xlarge"
     elif "autogluon" in image and "gpu" in image:
         return "p3.2xlarge"
     elif "trcomp" in image:
         return "p3.2xlarge"
     return "p3.8xlarge" if "gpu" in image else "c5.18xlarge"
 
+def assign_sagemaker_ami(image, region):
```

Comment:
Please clarify that this is the AMI for local tests.

Perhaps
```
def assign_sagemaker_local_test_ami(...)
```
?

Reply:
Updated